src/shared/renderUtils.jsx

```diff
@@ -1,19 +1,60 @@
 import React from "react";
 
-function renderBody(body){
+function renderBody(body, scope = "", className = ""){
 	const paragraphs = body.split("\n"); // splits the string at every line-break
-	return paragraphs.map(renderParagraph);
+
+	return paragraphs.map(
+		renderParagraph.bind(this, scope, className)
+	);
 }// renderBody
 
-function renderParagraph(text, index){
-	const key = `about__body__paragraph_${index}`;
+function renderParagraph(scope, className, text, index){
+	const key = `${scope}__body__paragraph_${index}`;
+	
+	//if there is a link inside the text, cut it out
+	const link = findLink(text) || {};
+	if(link) text = text.slice(0, link.index);
```

Comment:
This is neat, but it's gonna trip up when there are multiple links, so one to be aware of moving forward; although we don't need to address it just yet.